### ISSUE TITLE
[BUG] Clickhouse as destination doesn't work

### ISSUE THREAD

**Issue by drozdovilyaa** [mage-ai/mage-ai#4480]
### Mage version

v0.9.61

### Describe the bug

While testing the connection to ClickHouse as the destination in the data integration pipeline, I am getting the error:

```
Failed to connect, see error below.

type object 'ClickHouseDialect_http' has no attribute 'dbapi'
```

Connection example:

```
sqlalchemy_url: clickhouse://default:password@192.168.0.10:8123/db_name
table_name: table_name
```

### Screenshots

![image](https://github.com/mage-ai/mage-ai/assets/63257335/bb1c2383-cf94-4a63-aa7b-37308a0658dd)

**Comment by senthilimc** [comment]
@drozdovilyaa did you solve this error? am having the same issue

**Comment by wangxiaoyou1993** [comment]
The fix is included in version 0.9.68
